client/containers/Pub/PubHeader/Review/review.scss

```diff
@@ -0,0 +1,29 @@
+.review-component {
+	max-width: fit-content;
+	position: absolute;
+	right: 0;
+	top: 100%;
+
+	.review-button {
+		padding-top: 18px;
+	}
+	.review-submission-button {
+		.bp3-button {
+			color: white;
+			.bp3-icon {
+				color: white !important;
+			}
+		}
+		.bp3-button:hover {
+			color: white;
+			.bp3-icon {
+				color: white !important;
+			}
+		}
```

Comment:
Small suggestion, you could use nesting here w/ `&:hover` just so you don't write the `.bp3-button` and `.bp3-icon` selectors twice.